src/main/features/core/tray.js

```diff
@@ -138,7 +142,9 @@ Emitter.on('audiooutput:list', (event, devices) => {
       });
     }
   });
-  setContextMenu();
+  if (appIcon) {
```

Comment:
Ah yeah, sorry, I just say a lot and commented on the one that was on screen :+1: